### PR TITLE
Issue #92 show attachments in a popup

### PR DESCRIPTION
**Pull request by JLTRY**
implement Issue #92
correct also language $lang->load('plg_attachments_framework')

### CODE REVIEW

**Comment by JLTRY** [comment]
@parapente to be reviewed

File: attachments_component/admin/config.xml [hunk 1]
```diff
@@ -188,11 +188,12 @@
            label="ATTACH_CSS_STYLE_FOR_ATTACHMENTS_TABLES" size="40"
            description="ATTACH_CSS_STYLE_FOR_ATTACHMENTS_TABLES_DESCRIPTION">
     </field>
-    <field name="file_link_open_mode" type="radio" default="in_same_window" layout="joomla.form.field.radio.switcher"
+    <field name="file_link_open_mode" type="radio" default="in_same_window" class="btn-group btn-group-yesno"
            label="ATTACH_FILE_LINK_OPEN_MODE"
```
Comment (parapente) [inline attachments_component/admin/config.xml:192]: I believe it is better presented as a list. Radio button groups are not very good at presenting a lot of text in smaller screens.
```suggestion
    <field name="file_link_open_mode" type="list" default="in_same_window"
```

**Review by parapente — COMMENTED** [review]
The code seems to be ok but it still doesn't work in a test site I have locally on my computer. I will continue testing tomorrow at work.

**Comment by parapente** [comment]
@JLTRY it seems that the bootstrap modal wasn't loaded and that is why it wasn't working. I believe I have fixed it both for the frontend and for the backend (show in editor plugin was using the same code).

There is a problem with mobile view as the modal doesn't seem to work there. Maybe we should check the view and disable it when we detect mobile view. 

Also I have added a check for using the modal only on files that the browser can show (*.pdf and image/* from mimetype). The rest are treated as simple download links. Tell me what you think.

**Comment by parapente** [comment]
The latest patch I added disables modals for mobiles so the files would appear for download immediately.

**Comment by JLTRY** [comment]
@parapente modified iframe size that was too small

**Comment by parapente** [comment]
@JLTRY bodyHeight and modalWidth should be 80 (it is 80vh and 80vw accordingly so 80% of the height and 80% of the width of the screen). Using pixels is not ideal as on large screens the available area is too small and on smaller screens it is too large. You can see the result [here](https://srv-dide-v.thess.sch.gr/portal/).

**Comment by parapente** [comment]
Could it be that the template of your site don't define .modal-dialog.jviewport-width80 { width: 80vw; max-width:none } and .jviewport-height80 { height: 80vh; } ?

**Comment by JLTRY** [comment]
The style should be independent from the template
+ I re add the iframe height/width else it does not work

**Comment by JLTRY** [comment]
@parapente did you have time to test my latest commit ?

**Comment by parapente** [comment]
@JLTRY sorry I thought I had submitted the review but it was still pending. Merry Christmas!

**Comment by JLTRY** [comment]
@parapente Merry Christmas to you !!!

**Pull request merged**